ereuse_tag/view.py

```diff
@@ -41,3 +44,13 @@ def post(self):
         response = jsonify(ids)
         response.status_code = 201
         return response
+
+
+class VersionView(View):
+    def get(self, *args, **kwargs):
+        """Get version."""
+
+        with open("ereuse_tag/__init__.py", encoding="utf8") as f:
+            version = re.search(r'__version__ = "(.*?)"', f.read()).group(1)
```

Comment:
Why reading `ereuse_tag/__init__.py` as a plain text file instead of as a python module. e.g.:
```python
from . import __version__
```
Sample on [flask source code](https://github.com/pallets/flask/blob/5bfe236fb531045497396bc85ee5026c6c9c68eb/src/flask/cli.py#L282)

Reply:
This is a very great idea. I replicate this in all places where I use the open __init__ file in tag and dh